style.css

```diff
@@ -0,0 +1,286 @@
+}
+
+main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 50px 24px;
+  gap: 5px;
+  background-image: url(images/top-circles.png);
+  background-repeat: no-repeat;
+  width: 375px;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 280px;
+}
+
+.toolbar img {
+  width: 35px;
+  height: 35px;
+}
+
+.toolbar .logo {
+  margin-top: 10px;
+}
+
+.headline {
+  color: #fff;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 44px;
+}
+
+.headline h1 {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 31px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: #7d73fc;
+  margin-bottom: 5px;
+  margin-top: 38px;
+  padding: 0;
+}
+
+.headline h4 {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 24px;
+  margin-bottom: 5px;
+}
+
+.intro-text {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  flex-direction: row;
+  width: 365px;
+  padding: 10px;
+}
+
+.intro-text p {
+  width: 327px;
+  height: 170px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #c8c8d4;
+}
+
+.about-services {
+  display: flex;
+  justify-content: left;
+  align-items: flex-start;
+  flex-direction: row;
+  width: 365px;
+  margin-top: 0;
+  margin-left: 15px;
+  margin-right: 15px;
+}
+
+.about-services2 {
+  display: flex;
+  justify-content: left;
+  align-items: flex-start;
+  flex-direction: row;
+  width: 365px;
+  margin-top: 0;
+  margin-left: 25px;
+  margin-right: 15px;
+}
+
+.service-list {
+  display: flex;
+  flex-direction: column;
+  margin: 10px 10px;
+  padding: 15px;
+  justify-content: left;
+  align-items: center;
+}
+
+hr {
+  width: 100%;
+}
+
+.service-list ul li {
+  list-style: none;
+  margin: 5px 0;
+  text-align: left;
+  font-size: 20px;
+}
+
+.service-list ul li a {
+  text-decoration: none;
+  color: #f5f5ff;
+}
+
+.service-list .title {
+  width: 100%;
+  height: 50px;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+}
+
+.service-list .title h6 {
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+  color: #9490ff;
+}
+
+.social-media ul {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  gap: 1rem;
+  margin: 20px 10px;
+}
+
+.works-header {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  flex-direction: column;
+  color: #7d73fc;
+  margin: 0 10px 0 10px;
+  padding: 15px 15px 0 15px;
+  width: 375px;
+}
+
+.works {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 375px;
+  margin: 10px 10px;
+  padding: 15px;
+  background-image: url(images/top-circles.png);
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] The main container of the projects should be aligned with `display: grid` instead of the flexbox. I suggest you fix that to get your project approved since this is a core requirement of the project.